src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2838,7 +2837,8 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
                 "version_detected": "%s",
                 "runtime_version": "%s"
                 }
-                """ % (name, plan, rg_name, sku, os_name, loc, _src_path_escaped, detected_version, runtime_version)
+                """ % (name, plan, rg_name, get_sku_name(sku), os_name, loc, _src_path_escaped, detected_version,
+                       runtime_version)
```

Comment:
Why do you switch from sku value to sku name

Reply:
Same comment as above.